.ci.yaml

```diff
@@ -1,17 +1,40 @@
+# Describes the targets run in continuous integration environment.
+#
+# Flutter infra uses this file to generate a checklist of tasks to be performed
+# for every commit.
+#
+# More information at:
+#  * https://github.com/flutter/cocoon/blob/master/CI_YAML.md
 enabled_branches:
   - master
 
+platform_properties:
+  linux:
+    properties:
+      caches: "[{\"name\": \"dart_pub_cache\", \"path\": \".pub_cache\"}]"
```

Comment:
@christopherfujino I updated this to be json encoded. For repos with more caches, we can use a yaml multiline block to make this prettier